### PR TITLE
feat: add i18n for the Polish language

### LINKED ISSUE #583: [i18n] add Polish(pl) language
Hey!
I can provide myself to do these translations.

### PR DESCRIPTION
**Pull request by gbaranski**
closes #583

### CODE REVIEW

**Comment by fabian-hiller** [comment]
Do you know another native speaking developer that could review the translations for me?

**Comment by gbaranski** [comment]
> Do you know another native speaking developer that could review the translations for me?

Well, @stanbar may give a helpful hand here. I'll ask him.

File: packages/i18n/src/pl.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
+import { Language } from './types';
+
+// prettier-ignore
+const language: Language = {
+  code:             'pl',
+  schema:           (issue) => `Nieprawidłowy typ: Oczekiwano ${issue.expected}, ale otrzymano ${issue.received}`,
+  specific: {
+    bic:            (issue) => `Nieprawidłowy BIC: Otrzymano ${issue.received}`,
+    bytes:          (issue) => `Nieprawidłowe bajty: Oczekiwano ${issue.expected}, ale otrzymano ${issue.received}`,
+    creditCard:     (issue) => `Nieprawidłowa karta kredytowa: Otrzymano ${issue.received}`,
```
Comment (stanbar) [inline packages/i18n/src/pl.ts:10]: I would change to `Nieprawidłowy numer karty kredytowej: ...`

File: packages/i18n/src/pl.ts [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    cuid2:          (issue) => `Nieprawidłowy Cuid2: Otrzymano ${issue.received}`,
+    custom:         (issue) => `Nieprawidłowe dane: Otrzymano ${issue.received}`,
+    customAsync:    (issue) => `Nieprawidłowe dane: Otrzymano ${issue.received}`,
+    decimal:        (issue) => `Nieprawidłowa wartość dziesiętna: Otrzymano ${issue.received}`,
+    email:          (issue) => `Nieprawidłowy email: Otrzymano ${issue.received}`,
+    emoji:          (issue) => `Nieprawidłowa emoji: Otrzymano ${issue.received}`,
+    endsWith:       (issue) => `Nieprawidłowe zakończenie: Oczekiwano ${issue.expected}, ale otrzymano ${issue.received}`,
```
Comment (stanbar) [inline packages/i18n/src/pl.ts:17]: More nativeish would be "Nieprawidłowa końcówka ...."

**Review by stanbar — APPROVED** [review]
Small suggestions, besides that, lgtm

**Comment by fabian-hiller** [comment]
I will review and merge it after @gbaranski responds to your feedback.

**Comment by fabian-hiller** [comment]
@gbaranski can you have a look at the comments?

File: packages/i18n/src/pl.ts (hunk 1 above)
Comment (gbaranski) [inline packages/i18n/src/pl.ts:10]:
```suggestion
    creditCard:     (issue) => `Nieprawidłowy numer karty kredytowej: Otrzymano ${issue.received}`,
```

File: packages/i18n/src/pl.ts (hunk 2 above)
Comment (gbaranski) [inline packages/i18n/src/pl.ts:17]:
```suggestion
    endsWith:       (issue) => `Nieprawidłowa końcówka: Oczekiwano ${issue.expected}, ale otrzymano ${issue.received}`,
```

**Review by gbaranski — COMMENTED** [review]
applying suggestions based on the comments by @stanbar

**Comment by gbaranski** [comment]
done @fabian-hiller

**Comment by fabian-hiller** [comment]
Thank you! I will merge it soon!

**Pull request merged**